### DISCUSSION TITLE
Possible to return resting shapes to original position after repulsing on mouse hover

### DISCUSSION THREAD

**Discussion by stylessard** [matteobruni/tsparticles#2405] [Q&A] [answered]
Hi, I've been toying around with tsParticles and loving it. I have a quick question regarding an interaction I'm trying to implement.

I'd like to add a couple of immobile (to start) shapes to the middle of the canvas, which are repulsed when hovered over, but return to their original positions afterwards. Is that possible?

Thank you!

**Comment by matteobruni** [comment, marked as answer]
It's not actually a thing, you can request this feature in the [issues](https://github.com/matteobruni/tsparticles/issues)
